src/EnergyPlus/DXCoils.cc

```diff
@@ -5224,7 +5225,8 @@ namespace DXCoils {
 				MySizeFlag( DXCoilNum ) = false;
 			}
 
-			DXCoil( DXCoilNum ).RatedCBF( 1 ) = CalcCBF( DXCoil( DXCoilNum ).DXCoilType, DXCoil( DXCoilNum ).Name, DXCoil( DXCoilNum ).RatedInletDBTemp, HPInletAirHumRat, DXCoil( DXCoilNum ).RatedTotCap( 1 ), DXCoil( DXCoilNum ).RatedAirMassFlowRate( 1 ), DXCoil( DXCoilNum ).RatedSHR( 1 ) );
+			Real64 const RatedAirMassFlowRateSeaLevel = DXCoil( DXCoilNum ).RatedAirVolFlowRate( 1 ) * PsyRhoAirFnPbTdbW( StdPressureSeaLevel, DXCoil( DXCoilNum ).RatedInletDBTemp, HPInletAirHumRat, RoutineName );
+			DXCoil( DXCoilNum ).RatedCBF( 1 ) = CalcCBF( DXCoil( DXCoilNum ).DXCoilType, DXCoil( DXCoilNum ).Name, DXCoil( DXCoilNum ).RatedInletDBTemp, HPInletAirHumRat, DXCoil( DXCoilNum ).RatedTotCap( 1 ), RatedAirMassFlowRateSeaLevel, DXCoil( DXCoilNum ).RatedSHR( 1 ), StdPressureSeaLevel );
```

Comment:
@rraustad 

First, I'd like to define a couple of the variables that aren't immediately clear:

 - `StdBaroPress` - Standard "atmospheric pressure" based on elevation (ASHRAE HOF p6.1) defined [here](https://github.com/NREL/EnergyPlus/blob/8489490953551511afd4898eea8dde65daa7e906/src/EnergyPlus/DataEnvironment.cc#L132) and [calculated here](https://github.com/NREL/EnergyPlus/blob/8489490953551511afd4898eea8dde65daa7e906/src/EnergyPlus/WeatherManager.cc#L4635) based on the elevation of the current simulation.
 - `StdPressureSeaLevel` - One atmosphere at sea level. The standard pressure part of "standard temperature and pressure". 101.325 kPa. Constant. 

The problem was that when calculating the *rated* coil bypass factor for `Coil:WaterHeating:AirToWaterHeatPump:*`, it was using all of the rated conditions except for the atmospheric pressure. It was using `StdBaroPress` instead, which is incorrect. `CalcCBF` had that hard coded in. The point of these changes was to open up `CalcCBF` to accept a different pressure--`StdPressureSeaLevel`--when I wanted to pass that in (on this line) and to default it to `StdBaroPress` at all other times. This changes the calculation of `CalcCBF` for `Coil:WaterHeating:AitToWaterHeatPump:*` objects only and leaves the calculation for all other DX Coils the same as before assuming they are correctly handling calculation of rated conditions.